Formula/dosfstools.rb

```diff
@@ -0,0 +1,41 @@
+class Dosfstools < Formula
+  desc "Tools to create, check and label file systems of the FAT family"
+  homepage "https://github.com/dosfstools"
+
+  head "https://github.com/dosfstools/dosfstools.git"
+
+  stable do
+    url "https://github.com/dosfstools/dosfstools/releases/download/v4.0/dosfstools-4.0.tar.gz"
+    mirror "https://fossies.org/linux/misc/dosfstools-4.0.tar.gz"
+    sha256 "25809c30403c9a945ae34827ec75df32ff55017415a720864fefccc8c8f9991f"
+
+    patch do
+      # This patch restores the old defaults in versions up to v3.0.28
```

Comment:
A minor style issue, but would be nice to put the comment before the `patch do` instead of inside the block.